publish/tests/test_version.py

```diff
@@ -1,8 +1,25 @@
 import pytest
 
+from publish.models import Version
 from .fuzzy import DandisetIdentifierRe, TimestampRe, VersionRe
 
 
+@pytest.mark.django_db
+def test_version_make_version_nosave(dandiset, version_factory):
```

Comment:
If I had to guess, it looks like this is a factoryboy fixture. Is it required for this test, or does it inject something somewhere?

Reply:
You're right, `version_factory` is unnecessary in this case.

Reply:
@dchiquit And yes, these fixtures are defined by https://pytest-factoryboy.readthedocs.io/ , which is small but very helpful.